### PR TITLE
[CI] CI for paco

### PR DESCRIPTION
**Pull request by minkiminki**

### CODE REVIEW

**Comment by SkySkimmer** [comment]
Does upstream test coq master in its CI?

**Comment by minkiminki** [comment]
I enabled its CI. It tests for the Coq master branch too.

File: dev/ci/ci-paco.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download paco
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+( cd "${CI_BUILD_DIR}/paco"
+  cd src
```
Comment (Alizter) [inline dev/ci/ci-paco.sh:13]:
```suggestion
( cd "${CI_BUILD_DIR}/paco/src"
```

**Review by Alizter — APPROVED** [review]

**Comment by Alizter** [comment]
@minkiminki Can you squash those commits? I'll merge when the CI finishes after.

**Comment by minkiminki** [comment]
squashed

**Comment by SkySkimmer** [comment]
What's the basis for adding this in CI? Did paco hit interesting bugs in the past?

**Comment by minkiminki** [comment]
@SkySkimmer As far as I know, there has been no such case.

My main motivation for this PR is to add [Interaction Trees](https://github.com/DeepSpec/InteractionTrees) to CI (@Lysxia seems to be interested in this), and ITree depends on paco.

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**